### PR TITLE
Assessment point system

### PR DESCRIPTION
**Pull request by gbilgedal**
created a pointing system according to hamilton depression rating scale

### CODE REVIEW

File: src/containers/PsychoAssess/sections/Questions.jsx [hunk 1]
```diff
@@ -1,16 +1,119 @@
...
+    for (let i = 0; i < radioButtons.length; i++) {
+      if (radioButtons.item(i).checked) {
+        userScore += parseInt(radioButtons.item(i).value);
+        checkedRadioNumber++;
+      }
+    }
+    if (checkedRadioNumber === QUESTION_QUANTITY) {
+      return userScore;
+    }
+    return -15;
```
Comment (louisrli) [inline src/containers/PsychoAssess/sections/Questions.jsx]: please no magic numbers, -15 is in here hard-coded twice. i have no idea what -15 means

File: src/containers/PsychoAssess/sections/Questions.jsx [hunk 2]
```diff
@@ -1,16 +1,119 @@
...
+            {t("assessment.closeButton")}
+          </Button>
+        </Modal.Footer>
+      </Modal>
+    );
+  };
+
+  const showResults = (score) => {
+    if (score !== -15) {
+      if (score >= 52) {
```
Comment (louisrli) [inline src/containers/PsychoAssess/sections/Questions.jsx]: there should be 0 magic numbers in this function
```
if (score >= Scores.VERY_SEVERE) {
} else if (score >= Scores.SOME_OTHER_TIER) {
} else if (...)
```

**Review by louisrli — CHANGES_REQUESTED** [review]

File: src/containers/PsychoAssess/sections/Questions.jsx (hunk 1 above)
Comment (gbilgedal) [reply src/containers/PsychoAssess/sections/Questions.jsx]: done, please take a look when available.

File: src/containers/PsychoAssess/sections/Questions.jsx (hunk 2 above)
Comment (gbilgedal) [reply src/containers/PsychoAssess/sections/Questions.jsx]: done, please take a look when available.

File: src/containers/PsychoAssess/sections/Questions.jsx (hunk 1 above)
Comment (gbilgedal) [reply src/containers/PsychoAssess/sections/Questions.jsx]: done

File: src/containers/PsychoAssess/sections/Questions.jsx (hunk 2 above)
Comment (gbilgedal) [reply src/containers/PsychoAssess/sections/Questions.jsx]: done, please take a look when available.

**Review by ttugba — APPROVED** [review]

**Review by basil-kawak — COMMENTED** [review]
good job @gbilgedal

**Review by louisrli — APPROVED** [review]

**Pull request merged**